perses/rjmc/topology_proposal.py

```diff
@@ -300,38 +301,79 @@ class AtomMapper(object):
     atom_expr
     bond_expr
     allow_ring_breaking
+    map_strategy
 
     """
 
+    def _score_maps(mol_A, mol_B, maps):
+        """ Gives a score for how well each map in a list
+        recapitulates the geometry of ligand B.
+        If the geometry of ligand B is known, it can identify the closest map,
+        if it's not known, it can still be helpful as maps with the same score
+        are redundant --- i.e. a flipped phenyl ring.
+
+
+        Parameters
+        ----------
+        mol_A : oechem.oemol
+            old molecule
```

Comment:
Both A and B must have geometry, right?
If you they don't have geometry, can you do this over many possible conformers and take the minimum?

Reply:
So it depends on the map_strategy. If we use 'core' which is the default, then it doesn't actually matter what the geometry of the molecules are, because the same map will be returned anyway.

Using the 'geometry' is only helpful in the case of a constrained active site where the orientation isn't free. Generating lots of possible conformers in vacuum and finding the lowest energy one might not  actually capture this necissarily and help in the way that we want it to.